src/Orleans/Messaging/Message.cs

```diff
@@ -1106,6 +1127,9 @@ public static void Serializer(object untypedInput, ISerializationContext context
                 if ((headers & Headers.IS_NEW_PLACEMENT) != Headers.NONE)
                     writer.Write(input.IsNewPlacement);
 
+                if ((headers & Headers.IS_USING_IFACE_VERSION) != Headers.NONE)
```

Comment:
No need to change it, but this is a bit redundant: the flag is only set in headers if the bool is true, so we don't need to waste a byte & a read/write on it.